spark/src/main/scala/geotrellis/spark/Implicits.scala

```diff
@@ -65,7 +65,6 @@ trait Implicits
     with split.Implicits
     with stitch.Implicits
     with summary.Implicits
-    with summary.polygonal.Implicits
     with tiling.Implicits
```

Comment:
@CloudNiner The implicit was here to allow using the functionality with `import geotrellis.spark._` and keep the number of imports required to use GeoTrellis functionality down. Is this intentional move that was talked about that just goes against that?

Reply:
This one was removed because there no longer is a `geotrellis.spark.summary.polygonal.Implicts`. That entire file was only used by the old polygonal summary methods so it was removed.